lib/validator.js

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const validation = require('bedrock-validation');
+
+exports.validator = ({bodySchema, querySchema}) => {
+  return (req, res, next) => {
+    if(querySchema) {
+      const result = validation.validateInstance(req.query, querySchema);
+      if(!result.valid) {
+        return next(result.error);
+      }
+    }
+    if(bodySchema) {
```

Comment:
Since `bodySchema` is the more common use case, please move this above the `querySchema` conditional on L10.  That will save a comparison in the common case.